Monitor/Monitor.config

```diff
@@ -203,3 +203,53 @@ if(PLUGIN_MONITOR_WEBKITBROWSER_RESIDENT_APP)
     map_append(${configuration} observables ___array___)
     map_append(${configuration} observables ${RESIDENT_APP_MONITOR_CONFIG})
 endif()
+
+if(PLUGIN_MONITOR_INSTANCES_LIST)
+
+    # 'PLUGIN_MONITOR_INSTANCES_LIST' contains a semi-colon (';') separated list of Monitor observable
+    # instances. In turn, each instance is a comma (',') separated list defining an observable instance
+    # element ("callsign,memory,memorylimit,operational,window,limit"), where:
+    # - "callsign": The callsign/name of the plugin to be monitored.
+    # - "memory": Interval in seconds used for checking the plugin's memory usage.
+    #    If 0, means don't check memory.
+    # - "memorylimit": Value in KiB above which memory usage is considered excessive.
+    #    Ignored if "memory" is zero.
+    # - "operational": Interval in seconds used for checking plugin's operational state.
+    # - "window": Interval in seconds considered for callsign restart throttling.
+    # - "limit": Maximum number of times the plugin can be restarted within the "window" time.
+    # More details of the parameters is available at https://github.com/rdkcentral/rdkservices/tree/sprint/2101/Monitor/doc
+
+    map_append(${configuration} observables ___array___)
+    foreach(instance ${PLUGIN_MONITOR_INSTANCES_LIST})
+        map()
```

Comment:
I think the ``` map()``` should move after the ```list(GET instance_conf 5 limit)```. This keeps all the JSON stuff tighter together.